### DISCUSSION TITLE
Licensing Recommendation

### DISCUSSION THREAD

**Discussion by jcboysha** [GirkovArpa/transtube#1] [General]
The Non-violent and Cooperative Non-violent Public Licenses may do what you want them to do for this project. They can be found at https://thufie.lain.haus/NPL.html. 

But that's just like, my opinion.

**Comment by GirkovArpa** [comment]
Thank you for the recommendation.  But as a derivative work of [veadotube-mini](https://olmewe.itch.io/veadotube-mini) I believe it should inherit the license and [terms of use](https://veado.tube/terms/).
